### PR TITLE
added hash

### PR DESCRIPTION
**Pull request by shakeib98**
### What does this PR do?

added hash in cryptography category

Closes # ...

### CODE REVIEW

File: resources/master_term_bank_en.csv [hunk 1]
```diff
@@ -20,4 +20,5 @@ application/json,dao,"An acronym for decentralized autonomous organizations.  DA
 application/json,glosseta,"Glosseta is a open-source web3 glossary meant to help those onboarding into web3 understand the terms/lingo floating around the internet.  Glosseta is meant to be your companion on your adventure from web2 into web3 and to help you on your path of learning. We're all going to make it (WAGMI)!",en,GLOSSETA-PROD,application
 application/json,probably nothing,"Definitely something.",en,GLOSSETA-PROD,general
 application/json,gas,"Gas refers to the unit that measures the amount of computational effort required to execute specific operations on the Ethereum network. Since each Ethereum transaction requires computational resources to execute, each transaction requires a fee. Gas refers to the fee required to conduct a transaction on Ethereum successfully.",en,GLOSSETA-PROD,general
-application/json,irl,"An acronym for in real life.",en,GLOSSETA-PROD,general
\ No newline at end of file
+application/json,irl,"An acronym for in real life.",en,GLOSSETA-PROD,general
+application/json,hash,"Hashing is simply passing some data through a formula that produces a result, called a hash. That hash is usually a string of characters.",en,GLOSSETA-PROD,cryptography
```
Comment (narbs91) [inline resources/master_term_bank_en.csv:24]: Slight recommendation to switch up the wording a bit and add a bit more detail:

`A hash is the result of a deterministic transformation of data after being passed through a mathematical formula. A hash is typically a long series of alphanumerical characters having the unique property of being very hard to reverse engineer. That is it's very difficult to figure out the original data from just the hash itself.  This is one of the foundational components enabling the security and privacy of blockchains.`

**Review by narbs91 — CHANGES_REQUESTED** [review]
Thank you for taking the time time to add this definition @shakeib98 !  A slight suggestion around changing some wording and adding some more details.

Also just a reminder to please open up a issue template for a definition intake so that we can keep track of each request

File: resources/master_term_bank_en.csv (hunk 1 above)
Comment (shakeib98) [reply resources/master_term_bank_en.csv:24]: Alright!

Comment (narbs91) [reply resources/master_term_bank_en.csv:24]: Hi @shakeib98 just checking up, how are you doing on this issue?

**Comment by narbs91** [comment]
Closing issue as #39 takes care of original intent

**Comment by narbs91** [comment]
@all-contributors please add @shakeib98 for data

**Pull request closed without merging**